client/src/components/EventTypesTab.js

```diff
@@ -213,7 +213,7 @@ export default function EventTypesTab(props) {
                     </Grid>
                     <Grid item sm={6} align="right"
                         className={classes.newEventBtnContainer}>
-                        <EventTypeDialog />
+                        <EventTypeDialog setCardInfo={setCardInfo} userUrl={userUrl} />
```

Comment:
this will finally rerender the cards in eventType Tab when a new card created, so we don't need to refresh the page mannually